docs/shared-content/installation/server-downloads.include.md

```diff
@@ -1,15 +1,8 @@
-### Download the long-term support Octopus Server {#slow-lane}
+### Download the Octopus Server
 
-The latest long-term support version of Octopus Deploy:
+The latest version of the Octopus Deploy Server:
 
-- [Octopus Server x64](https://octopus.com/downloads/slowlane/WindowsX64/OctopusServer)
-- [Octopus Server 32-bit/x86](https://octopus.com/downloads/slowlane/WindowsX86/OctopusServer)
+- [Octopus Server](https://octopus.com/downloads/slowlane/WindowsX64/OctopusServer)
 
-If you need an older version of a long-term support release, visit the [previous downloads page](https://octopus.com/downloads/previous) to locate the specific version you need.
+If you need an older version of the Octopus Deploy Server, visit the [previous downloads page](https://octopus.com/downloads/previous) to locate the specific version you need.
```

Comment:
@michaelnoonan any thoughts on previous versions of Octopus Server?

Reply:
Yep, that's correct. How does this fit?
```suggestion
We recommend using the [latest version](https://octopus.com/downloads) of the Octopus Deploy Server. If you need an older version of the Octopus Deploy Server, visit the [previous downloads page](https://octopus.com/downloads/previous) to locate the specific version you need.
```

Reply:
Great!